src/components/TooltipContainer.ts

```diff
@@ -0,0 +1,128 @@
+import { /*CSSProperties,*/ Component, createElement } from "react";
+import * as ReactTooltip from "react-tooltip";
+import "../ui/Tooltip.scss";
+
+export interface WrapperProps {
+    class: string;
+    mxObject: mendix.lib.MxObject;
+    mxform: mxui.lib.form._FormBase;
+    style: string;
+    readOnly: boolean;
+    friendlyId: string;
+    tooltipText: string;
+    linktext: string;
+    callMicroflow?: string;
+    callNanoflow?: Nanoflow;
+    tooltipPosition: "top" | "right" | "bottom" | "left";
+    bootstrapStyle: "dark" | "success" | "warning" | "error" | "info" | "light";
+    bootstrapEffect: "float" | "solid";
+    imageAttribute: string;
+    tooltipType: "linkTooltip" | "imageTooltip" | "webpageTooltip";
+    imageUrl: string;
+    imageTooltip: string;
+    reference: string;
+    websiteURL: string;
+}
+
+interface Nanoflow {
+    nanoflow: any[];
+    paramsSpec: {
+        Progress: string;
+    };
+}
+
+export default class TooltipContainer extends Component<WrapperProps> {
+
+    setTooltipType() {
+        if (this.props.tooltipType === "linkTooltip" && this.props.tooltipText && this.props.linktext) {
+            return createElement("div", { className: "widget" },
+                createElement("a", {
+                    "data-tip": this.props.tooltipText,
+                    "data-place": this.props.tooltipPosition,
+                    "data-type": this.props.bootstrapStyle,
+                    "data-effect": this.props.bootstrapEffect,
+                    "class": "linktext"
+                    }, this.props.linktext),
+                createElement(ReactTooltip, {})
+            );
+        } else if (this.props.tooltipType === "imageTooltip" && this.props.imageUrl && this.props.imageTooltip) {
+            return createElement("div", { className: "widget" },
+            createElement("img", {
+                "data-tip": this.props.imageUrl,
+                "data-place": this.props.tooltipPosition,
+                "data-type": this.props.bootstrapStyle,
+                "data-effect": this.props.bootstrapEffect,
+                "class": "linktext",
+                "src": this.props.imageUrl
+                }),
+            createElement(ReactTooltip, {})
+        );
+        } else if (this.props.tooltipType === "webpageTooltip" && this.props.reference && this.props.websiteURL) {
+            return createElement("div", { className: "widget" },
+                        createElement("a", { href: this.props.websiteURL }, this.props.reference),
+                        createElement("div", { className: "box" },
+                            createElement("iframe", { src: this.props.websiteURL, className: "iFrame" })
+                        ),
+                    createElement(ReactTooltip, {})
+        );
+        } else {
```

Comment:
What should happen is the modeler made a mistake in the config, nothing will show. You could help him with proper error message in the UI with an `Alert` component